cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -629,8 +634,20 @@ and then restart CDAP.
 
      for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
      
-If you are upgrading a secure Hadoop cluster, see the notes above about
-:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+#. You should recompile and redeploy your application. This will allow you to see your old
+   run history, logs, and |---| if you migrated your old metrics with the 
+   :ref:`metric migration tool <>` |---| metrics.
+
+   **Note:** You will no longer be able to see your previous logs in the CDAP Console (UI). 
+   To access your previous logs, please see the section on downloading logs in the
+   :ref:`Logging HTTP RESTful API <http-restful-api-logging>`.
+     
+#. If you are upgrading a secure Hadoop cluster, see the notes above about
+   :ref:`configuring secure Hadoop <install-secure-hadoop>`.
+
+   To upgrade a secure cluster, you should use ``kinit``::
```

Comment:
this note should also be further up, as you run kinit before running the upgrade tool.